administrator-guides/federation/README.md

```diff
@@ -0,0 +1,88 @@
+# Federation
+
+## Introduction
+
+Federation allows communication with other Rocket.Chat servers.
```

Comment:
Can we please have a small introduction /  architecture section explaining things such as what the Hub is, why is DNS relevant to Federation and and how can servers "trust" one another to get Federation going  (can we just force impose Federatation to any other server anywhere ?  that is not clear at all)